BriefAssistant/ClientApp/app/models/BriefInfo.ts

```diff
@@ -2,25 +2,33 @@
 import { CircuitCourtCase } from "./CircuitCourtCase";
 
 export class BriefInfo {
-	circuitCourtCase: CircuitCourtCase;
-	appendixDocuments: string;
-	conclusion: string;
-	caseFactsStatement: string;
-	argument: string;
-	oralArgumentStatement: string;
-	publicationStatement: string;
-	issuesPresented: string;
-	appellant: Appellant;
+    circuitCourtCase: CircuitCourtCase;
+    appendixDocuments: string;
+    conclusion: string;
+    caseFactsStatement: string;
+    argument: string;
+    oralArgumentStatement: string;
+    publicationStatement: string;
+    issuesPresented: string;
+    appellant: Appellant;
+    topName: string;
+    bottomName: string;
+    topRole: string;
+    bottomRole: string;
 
-	constructor() {
-		this.circuitCourtCase = new CircuitCourtCase();
-		this.appendixDocuments = "";
-		this.conclusion = "";
-		this.caseFactsStatement = "";
-		this.argument = "";
-		this.oralArgumentStatement = "";
-		this.publicationStatement = "";
-		this.issuesPresented = "";
-		this.appellant = new Appellant();
-	}
+    constructor() {
+        this.circuitCourtCase = new CircuitCourtCase();
+        this.appendixDocuments = "";
+        this.conclusion = "";
+        this.caseFactsStatement = "";
+        this.argument = "";
+        this.oralArgumentStatement = "";
+        this.publicationStatement = "";
+        this.issuesPresented = "";
+        this.appellant = new Appellant();
+        this.topName = "";
+        this.bottomName = "";
+        this.topRole = "";
+        this.bottomRole = "";
```

Comment:
I don't think the topName, bottomName, topRole and bottomRole need to be here since the backend determines those.